charts/cluster-scanner/Makefile

```
@@ -0,0 +1,3 @@
+.PHONY: test
+test:
+	helm unittest -3 .
```

```suggestion
	helm unittest .
```

the flag `-3` is not supported with the unittest plugin (>=0.3.x)